bundles/org.openhab.binding.meater/src/main/java/org/openhab/binding/meater/internal/MeaterAuthenticationException.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.meater.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * {@link MeaterException} is used when there is an authentication exception with MEATER REST API.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+@NonNullByDefault
+public class MeaterAuthenticationException extends Exception {
```

Comment:
**Minor:** This could extend `MeaterException` instead of `Exception`. In this case callers could choose to catch only `MeaterException` if no specialized handling is needed.

Both exception classes could also be moved into an "exceptions" subfolder for keeping the structure a bit cleaner, especially in case more exceptions would be added.

Reply:
Fixed!